Formula/immortal.rb

```diff
@@ -1,10 +1,10 @@
-require "language/go"
-
 class Immortal < Formula
   desc "OS agnostic (*nix) cross-platform supervisor"
   homepage "https://immortal.run/"
-  url "https://github.com/immortal/immortal/archive/0.16.0.tar.gz"
-  sha256 "b540882641b904be956ac0cf3a4a9cda9c5ea709c24b68c77c3f08a1a13272b6"
+  url "https://github.com/immortal/immortal.git",
```

Comment:
why aren't we still using the tarball?

Reply:
In my first `dep` formula  (ssh-vault) I used the tag/revision, therefor I though it was better, besides  make things more easy since I could doit in one step instead of downloding the source, calculate the checksum and update the formula

Reply:
Unless the build actually breaks with the tarball, we generally want to avoid tag+revision. If you use bump-formula-pr, it will do the tarball download and checksum computation automatically.

Reply:
oh ok, I will give a try now

Reply:
done!, next time before creating formula will check more in detail the `bump-formula-pr`